pages/installation.md

```diff
@@ -133,17 +146,19 @@ Assuming that you have RabbitMQ install correctly (and added to PATH) and the `r
 
 ###### Create a user
 
+It should be noted that these commands should be run from inside the sbin of your installation of RabbitMQ Server.
+
 ```
-sudo rabbitmqctl add_user oop somepassword
-sudo rabbitmqctl list_permissions --vhost /
-sudo rabbitmqctl set_permissions -p / oop ".*" ".*" ".*"
+rabbitmqctl add_user oop somepassword
+rabbitmqctl list_permissions --vhost /
+rabbitmqctl set_permissions -p / oop ".*" ".*" ".*"
 ```
 
 Ensure this is reflected in `OOP_AMQP_ADDRESS` connection string environment variable.
 
 ###### Import the queue config
 
-There is a pre-configured queue config in the [oop-queue-config](https://github.com/open-interop/oop-queue-config) repository. Download `rabbit-config.json` from the repo and run `sudo rabbitmqadmin import rabbit-config.json` to import it.
+There is a pre-configured queue config in the [oop-queue-config](https://github.com/open-interop/oop-queue-config) repository. Download `rabbit-config.json` from the repo and run `rabbitmqadmin import rabbit-config.json` to import it.
```

Comment:
Again break this up so there is a distinction between unix-based and windows-based systems as one needs the sudo command and the other doesn't.